streams/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -325,7 +318,7 @@ private List<KeyValue<Long, Long>> getAllRecordPerKey(final Long key, final List
     }
 
     @ParameterizedTest
-    @ValueSource(strings = {StreamsConfig.AT_LEAST_ONCE, StreamsConfig.EXACTLY_ONCE_V2})
+    @ValueSource(strings = {StreamsConfig.AT_LEAST_ONCE})
     public void shouldBeAbleToPerformMultipleTransactions(final String eosConfig) throws Exception {
```

Comment:
Seems we can apply the same clean as above for this test, and remove the parameter completely?

There is also more tests below (eg `shouldNotViolateEosIfOneTaskFails` and others) which still use at-least-once, but should only run with EOS.